src/main/java/org/example/app/controller/TwitController.java

```diff
@@ -0,0 +1,81 @@
+package org.example.app.controller;
+
+import org.example.app.entity.Twit;
+import org.example.app.service.TwitService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+@Controller
+public class TwitController {
+
+    @Autowired
+    private TwitService twitService;
+
+    @GetMapping(value = "/")
+    public String home(Model model) {
+        model.addAttribute("title", "Simple Twitter");
+        model.addAttribute("message", "Simple Twitter");
+        return "welcomePage";
+    }
+
+
+    @GetMapping("/list-twits")
+    public String listBooks(
```

Comment:
Did you mean list**Twits**?

Reply:
fixed

Reply:
Seems like it hasn't been fixed. Can you please verify again? Probably some other comments as well.

Reply:
fixed 100%

Reply:
For some reason I still don't see changes that you made. In latest commit you modified schema only https://github.com/nhrin/solr_tasks/commit/5c1cb96c2399b7d8102bb7add971db91c77f0746

Reply:
Please, check tab "files changed".
![Screenshot from 2022-05-13 11-00-02](https://user-images.githubusercontent.com/57841399/168238820-67a44ef9-b1df-4b99-9c06-b63fba710166.png)

Reply:
Ok. Sorry. I see, that you modified the endpoint. Now name your method in the same manner (i.e. public String list**Twits**...)

Reply:
Done